src/components/Sidebar/Sidebar.stories.tsx

```diff
@@ -35,7 +37,23 @@ const SidebarMeta = {
 } as ComponentMeta<typeof Sidebar>
 
 const Template: ComponentStory<typeof Sidebar> = () => {
-  return <Sidebar />
+  return (
+    <LoginContext.Provider
+      value={{
+        logout: async () => {
+          return undefined
+        },
```

Comment:
we should probably change this to show a toast so that it's obvious in tests that the button click works

Reply:
Makes sense! Added in https://github.com/isomerpages/isomercms-frontend/pull/959/commits/0d5ab3ef3354bf5b23af228baaf53fcbecdd072e